internal/envoy/install.go

```diff
@@ -55,7 +55,7 @@ func InstallIfNeeded(o *globals.GlobalOpts, p, v string) (string, error) {
 		}
 
 		fmt.Fprintln(o.Out, "downloading", tarballURL) //nolint
-		if err = untarEnvoy(installPath, tarballURL, o.UserAgent); err != nil {
+		if err = untarEnvoy(installPath, tarballURL, p, v); err != nil { //nolint
```

Comment:
lint hates me.. it doesn't like `=` and also doesn't like `:=`. To avoid making randomly `e`, I'm punting //nolint